pkg/rancher-desktop/components/Help.vue

```diff
@@ -0,0 +1,104 @@
+<script lang="ts">
+import { shell } from 'electron';
+import Vue from 'vue';
+import { mapState } from 'vuex';
+
+export default Vue.extend({
+  name:       'help',
+  props:      {
+    defaultUrl: {
+      type:     String,
+      default:  null,
+    },
+    tooltip: {
+      type:    String,
+      default: null,
+    },
+  },
+  computed: {
+    ...mapState('help', ['url']),
+    pageUrl(): string {
+      return this.defaultUrl ?? this.url;
+    },
+  },
+  methods:  {
+    openUrl() {
+      if (this.pageUrl) {
+        shell.openExternal(this.pageUrl);
+      }
+    },
+  },
+});
+</script>
+
+<template>
+  <div class="help-button">
+    <button
+      v-tooltip="{
+        content: pageUrl ? tooltip : null,
+        placement: 'right'
+      }"
+      class="btn role-fab ripple"
```

Comment:
Please add a comment that "fab" stands for "Floating Action Button" (I had to search, don't want others to as well).

Reply:
Fixed.